project/ntut-xuan/README.rst

```diff
@@ -0,0 +1,198 @@
+PolygonDust
+===========
+
+Description
+-----------
+
+PolygonDust is an application to measure the area size of polygon.
+
+“Dust” means we trying to shred the image into lot of pieces and measure
+polygon size by these pieces.
+
+.. image:: https://media.discordapp.net/attachments/950048467294760990/1284862080050135071/image.png?ex=66e82c8f&is=66e6db0f&hm=8f52c071f46be06d7d6d014e8551087f495e2cf1332a784079c4805429e7aea3&=&format=webp&quality=lossless&width=2880&height=848
+
+Problem to Solve
+----------------
+
+In size measurement of ploygon, we have two methods can do it.
+
+-  Pixel-Method: Shred the image to lot of rectangles and calculate the
+   size by these rectangles.
+-  Formula-Method: Calculate the polygon by formula.
+   e.g. :math:`W \times H` for rectangle of :math:`r^2 \times \pi` for
+   circle.
+
+PolygonDust is used to solve size calculation by Pixel-Method. In
+polygon description, we use image instead formula, that may make the
+problem more easier to solve.
+
+Expect the process steps should be:
+
+-  `Gaussian
+   Blur <https://zh.wikipedia.org/zh-tw/%E9%AB%98%E6%96%AF%E6%A8%A1%E7%B3%8A>`__
+   to decrease the image noice.
```

Comment:
Why do you have noise if the input are exact polygons?

Reply:
After I think about it, it may not usually appear the noice. I will like to add noice handle on purposal just for some case that need to resize the image or the input image may already resized and distorted. It will create some "noice" in polygon bounder. I'll think another image process related numerical process for image.

Reply:
This sounds more complicated than what people usually do in 3 months.